new-pipeline/src/dist.js

```diff
@@ -788,10 +791,19 @@ function updateDateRange(callback, dates, updatedByUser, shouldUpdateRangebar) {
   }
 }
 
-function displayHistograms(histogramsList, dates, useTable, cumulative, trim) {
+function displayHistograms(histogramsList, dates, useTable, cumulative, trim, includeSpill) {
   cumulative = cumulative === undefined ? false : cumulative;
   trim = trim === undefined ? true : trim;
 
+  if(!includeSpill)// Hide the 'spill' related data
```

Comment:
Some style comments: 

The comment should be on its own line, above the code it is commenting, and be a complete sentence (starting with a capital letter, ending with a period)
The opening brace should be on the same line as the if
The keyword if should have a space after it.

```patch
-if(!includeSpill)// Hide the 'spill' related data
-{
+// Hide the 'spill' bucket if present.
+if (!includeSpill) {
```